vision/style_transfer/fast_neural_style/transformer_net.py

```diff
@@ -0,0 +1,99 @@
+import torch
```

Comment:
where is this file from? should include attribution at the top.

Reply:
It's from the original PyTorch Style Transfer repo. I will add attribution.